Shopipy.Persistence/Models/GiftCard.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Shopipy.Persistence.Models;
+
+public class GiftCard
+{
+    [Key]
+    public int GiftCardId { get; set; }
+
+    [ForeignKey("Business")]
+    public required int BusinessId { get; set; }
+
+    [ForeignKey("Category")]
```

Comment:
How categories are used?

Reply:
![image](https://github.com/user-attachments/assets/cf850416-cadb-4bd7-8478-2c602f587545)

Reply:
how does it help with buying giftcards?

Reply:
I see the possible point of having a product called giftcard with a bit of special handling

Reply:
but as in the message, what's the point of overthinking, just don't bother with how giftcards will be bought

Reply:
eh, must've misunderstood. fixed